scapy/sendrecv.py

```diff
@@ -354,6 +366,7 @@ def sendpfast(x, pps=None, mbps=None, realtime=None, loop=0, file_cache=False, i
             raise
     finally:
         os.unlink(f)
+        return stdout, stderr, argv
```

Comment:
Meh.
Used interactively, this will prompt to the console tons of ugly strings...
It should *at least* be optional (triggered by a param).

The best thing you could do is provide a built-in param which would auto-parse the results and extract basic infos, then return them if the param is set